test/e2e_tests/pageManager/webapp/pages/conversationList.page.ts

```diff
@@ -58,6 +61,11 @@ export class ConversationListPage {
     this.unblockConversationMenuButton = page.locator(
       `${selectById('btn-unblock')}${selectByDataAttribute('conversation-list-options-menu')}`,
     );
+    this.moveConversationButton = page.locator(selectById('btn-move-to'));
+    this.moveToMenu = page.locator('ul.ctx-menu', {
+      has: page.locator(selectById('btn-create-new-folder')),
+    });
+    this.createNewFolderButton = this.moveToMenu.locator(selectById('btn-create-new-folder'));
```

Comment:
Please try to use accessible locators where possible over (test-)ids. For example in this case:

```suggestion
    this.moveConversationButton = page.getByRole('menu').getByRole('button', {name: 'Move to'});
    this.moveToMenu = page.getByRole('menu');
    this.createNewFolderButton = this.moveToMenu.getByRole('button', {name: 'Create new folder'});
```